processors/base/normalizers/phases_variations.json

```diff
@@ -0,0 +1,245 @@
+{
+  "IIB": [
+    "Phase 2B"
+  ],
+  "Pilot study": [
+    "Other"
+  ],
+  "Phase 1": [
+    "Phase 1"
+  ],
+  "Phase III/IV": [
+    "Phase 3",
+    "Phase 4"
+  ],
+  "Diagnostic New Technique Clincal Study": [
+    "Other"
+  ],
+  "Phase I": [
+    "Phase 1"
+  ],
+  "II-III": [
+    "Phase 2",
+    "Phase 3"
+  ],
+  "Phase 3-4": [
+    "Phase 3",
+    "Phase 4"
+  ],
+  "Other": [
+    "Other"
+  ],
+  "IIa": [
+    "Phase 2A"
+  ],
+  "II (Phase II study)": [
+    "Phase 2"
+  ],
+  "IIb": [
+    "Phase 2B"
+  ],
+  "Not entered": [
+    "Not applicable"
+  ],
+  "IIIb": [
+    "Phase 3B"
+  ],
+  "I-IIA": [
+    "Phase 1",
+    "Phase 2A"
+  ],
+  "III (Phase III study)": [
+    "Phase 3"
+  ],
+  "Phase4": [
+    "Phase 4"
+  ],
+  "n\\a": [
+    "Not applicable"
+  ],
+  "Phase 2": [
+    "Phase 2",
+    "Phase 2"
```

Comment:
Are these repeating values a mistake? There are a few cases of this (e.g. on `Phase 3` and `Phase 1/Phase 2`)